tailscale/client.go

```diff
@@ -1096,6 +1096,59 @@ func (c *Client) RotateWebhookSecret(ctx context.Context, endpointID string) (*W
 	return &webhook, c.performRequest(req, &webhook)
 }
 
+const (
+	ContactAccount  ContactType = "account"
+	ContactSupport  ContactType = "support"
+	ContactSecurity ContactType = "security"
+)
+
+type (
+	// ContactType defines the type of contact.
+	ContactType string
+
+	// Contacts type defines the object returned when retrieving contacts.
+	Contacts struct {
+		Account  Contact `json:"account"`
+		Support  Contact `json:"support"`
+		Security Contact `json:"security"`
+	}
+
+	// Contact type defines the structure of an individual contact for the tailnet.
+	Contact struct {
+		Email             string `json:"email"`
+		FallbackEmail     string `json:"fallbackEmail,omitempty"`
+		NeedsVerification bool   `json:"needsVerification"`
+	}
+)
+
+// Contacts retieves the contact information for a tailnet.
+func (c *Client) Contacts(ctx context.Context) (*Contacts, error) {
+	const uriFmt = "/api/v2/tailnet/%s/contacts"
+
+	req, err := c.buildRequest(ctx, http.MethodGet, fmt.Sprintf(uriFmt, c.tailnetPathEscaped))
+	if err != nil {
+		return nil, err
+	}
+
+	var contacts Contacts
+	return &contacts, c.performRequest(req, &contacts)
+}
+
+// UpdateContact updates the email for the specified ContactType within the tailnet.
+// If the email address changes, the system will send a verification email to confirm the change.
+func (c *Client) UpdateContact(ctx context.Context, contactType ContactType, email string) error {
```

Comment:
Contacts may eventually contain other fields like phone, address, etc. Since this doesn't accept a full `Contact`, maybe rename to `UpdateContactEmail`. Otherwise, consider passing in a `Contact`.